.github/workflows/dart.yml

```diff
@@ -82,17 +87,18 @@ jobs:
         run: flutter test
 
   job4:
+    if: false
```

Comment:
Instead of having this conditional we have to remember to remove, can we change these jobs to have a required input we set when running the workflow?

Reply:
The workflow is triggered automatically, so we'd better not make it depend on inputs. I think the conditional flag is fine. We can easily notice which job is disabled. 
<img width="695" alt="Screenshot 2024-04-12 at 4 36 47 PM" src="https://github.com/adobe/aepsdk_flutter/assets/47394585/764540db-8bcd-4247-b3e3-0f0606b3b392">